src/context.ts

```diff
@@ -74,7 +74,7 @@ export class Context implements RenamedUpdate {
      * Used by some middleware to store information about how a certain string
      * or regular expression was matched.
      */
-    public match?: string | RegExpMatchArray;
+    public match: string | RegExpMatchArray | undefined;
```

Comment:
If I'm not mistaken, this can be reverted now.

Reply:
Because we no longer rely on a mapped type which required the field to always be present (even though it isn't).

Reply:
```suggestion
    public match?: string | RegExpMatchArray;
```

Reply:
This is correct